hack/lib/mesh_resources/kafka-service-entry.yaml

```diff
@@ -5,9 +5,6 @@ metadata:
 spec:
   hosts:
     - my-cluster-kafka-bootstrap.kafka
```

Comment:
so we end up w/ less resources?  
IMO generally a good pattern to use only one format of URLs for the same host